workers/unity/Assets/Gdk/Core/Systems/SpatialOSSendSystem.cs

```diff
@@ -73,6 +94,13 @@ protected override void OnUpdate()
             {
                 view.TranslationUnits[componentTypeIndex].SendCommands(connection);
             }
+
+            ticks++;
+            if (ticks == METRIC_TICKS_UPDATE)
+            {
+                ticks = 0;
+                connection.SendMetrics(metricComputedEverySend());
```

Comment:
Might be nice to do some average instead of one computation every few ticks. This will have a higher variance.

Reply:
I completely missed the doc, will post there sorry.

Reply:
Done!